flask_admin/model/widgets.py

```diff
@@ -109,7 +109,10 @@ def get_kwargs(self, subfield, kwargs):
         elif subfield.type == 'BooleanField':
             kwargs['data-type'] = 'select'
             # data-source = dropdown options
-            kwargs['data-source'] = {'': 'False', '1': 'True'}
+            kwargs['data-source'] = json.dumps([
+                {'value': '0', 'text': gettext('No')},
```

Comment:
better to use lazy_gettext here, like this: https://github.com/iurisilvio/flask-admin/blob/translate_editable_boolean/flask_admin/model/filters.py#L98